.scripts/localeValidate.js

```diff
@@ -55,7 +46,19 @@ const getDeepKeys = (obj) => {
 };
 
 const defaultPath = join(localeDir, 'en');
-const languages = getDirectories(localeDir);
+
+// locale directories, ommitting `en` - the langauge to check missing keys against.
+const languagesButEn = languages.filter((v) => v !== 'en');
+
+
+const para = (lng, index) => {
+  const p = lng[index].slice((lng[index].indexOf('.') + 1));
+  return p;
+};
+const paraMinus = (lng, index) => {
+  const p = lng[index - 1].slice((lng[index].indexOf('.') + 1));
+  return p;
+};
```

Comment:
We do not need curly braces + return p here:

```suggestion
const paraMinus = (lng, index) =>
  lng[index - 1].slice((lng[index].indexOf('.') + 1));
```